public/less/_user.less

```diff
@@ -20,6 +20,9 @@
                         padding: 10px;
                         list-style: none;
 
+                        overflow: scroll; // prevent contents from spilling over
```

Comment:
Should be more one of `auto` or `hidden` because on the default theme it force scrollbar on all items.

![image](https://user-images.githubusercontent.com/4052400/59808957-d1293d00-92fe-11e9-96cb-95cd22c8d5d2.png)

Reply:
Huh, I guess that's what I get for only testing it in Firefox. (: I see the scrollbars on everything in Safari (don't have Chrome).

Setting it to `overflow: auto;` removes the extraneous scrollbars, and allows the overflowing email to scroll like it's supposed to. I'll make a commit for that.